### PR TITLE
Fix critical error in readme.md

### PR DESCRIPTION
**Pull request by stackola**

### CODE REVIEW

**Comment by SusuKacangSoya** [comment]
This is a serious error that will leave a lot of users infuriated.

I vote - pull this fix as soon as possible!

**Comment by KevinTyrrell** [comment]
Please fix this issue. Master branch is unusable in its current state.

**Comment by tahercool1** [comment]
Pls fix Thx

**Comment by VictorioBerra** [comment]
Please add unit tests for this fix.

**Comment by jaesung2061** [comment]
Since the author is gonna be confused as hell I thought I'd add the link where all the reactions are coming from.

https://www.reddit.com/r/ProgrammerHumor/comments/8f6cnb/made_my_first_big_contribution_to_open_source/

**Comment by lilypit** [comment]
Thank god someone found this failure - how could QA have missed it?

**Review by faustinoaq — APPROVED** [review]

**Comment by Fallenstedt** [comment]
Repo is unusable. Please accept this MR

**Comment by ryaan-anthony** [comment]
+1

**Comment by ayudh37** [comment]
Google wants to know your location

**Comment by scott-old** [comment]
Please merge ASAP.

**Comment by drselump14** [comment]
+1

**Comment by danielriley06** [comment]
We’ve been using this in production for months and our legal team just brought this to my (the very, very important CTO) attention. Please merge this PR or I’ll be forced to report you to the GitHub grammar police.

**Review by trivikr — APPROVED** [review]

**Comment by TheFunkyMonk** [comment]
LGTM

**Comment by rvirji-va** [comment]
:shipit:

**Comment by fipppi** [comment]
+1

**Comment by gsarpy** [comment]
Hey guys, I built a JavaScript library as a solution to this bug. It's called Your.js

**Review by Wruczek — APPROVED** [review]

**Review by JoniVR — APPROVED** [review]

**Comment by ArthurGarnier** [comment]
This bug avoid to read the file completely, merge ASAP!!

**Review by opcecco — APPROVED** [review]

**Comment by tobiastornros** [comment]
Which font is that?

**Review by ayudh37 — APPROVED** [review]

**Review by AUTplayed — APPROVED** [review]

**Review by albertorestifo — APPROVED** [review]

**Comment by bencoveney** [comment]
Also might want to consider:
Theres => There's
Rhobust => Robust

**Comment by Russell-IO** [comment]
Critical issue

File: README.md [hunk 1]
```diff
@@ -93,7 +93,7 @@ Other popular BTC average modules require incredible amount of bandwidth at regu
 
 This module **does not include exchange API's that do not have web sockets**. Why? Theres a billion other modules that do something similar, and perhaps this is a wake up call to any exchange that does not offer rhobust websocket support.
```
Comment (prendradjaja) [inline README.md:94]: While we're at it, can we change "API's" to "APIs"? Thanks.

Comment (prendradjaja) [reply README.md:94]: Also: rhobust -> robust

File: README.md [hunk 2]
```diff
@@ -93,7 +93,7 @@ Other popular BTC average modules require incredible amount of bandwidth at regu
...
 
-Many developers would rather not be bothered reading through additional API documentation which, in many cases, is incomplete, hard to follow and usually lacking node.js examples. I did it for you! In node. Your welcome.
+Many developers would rather not be bothered reading through additional API documentation which, in many cases, is incomplete, hard to follow and usually lacking node.js examples. I did it for you! In node. You're welcome.
 
 ### Why Should I use websockets?
```
Comment (prendradjaja) [inline README.md:98]: Weird capitalization

**Review by prendradjaja — CHANGES_REQUESTED** [review]

**Review by GarethPW — APPROVED** [review]

**Comment by Seluj78** [comment]
LGTM :+1:

**Comment by jjhidalgar** [comment]
LGTM. Some automatic tests should be made before making to the master branch though.

**Review by QuickBASIC — APPROVED** [review]

**Review by Seluj78 — APPROVED** [review]

**Review by TKasekamp — APPROVED** [review]

**Comment by woolleyalan** [comment]
Is there a workaround for this for the time-being ?

**Review by redcap3000 — APPROVED** [review]
While this is no excuse; in my defense I was using a very terrible keyboard; it has been very recently laid to rest. I appreciate your contribution and encourage more! Welcome to the wonderful world of free and open source.

**Comment by nth-commit** [comment]
This is a showstopper for my organisation. My boss is really riding me hard on this one, I assured him that crypto-socket was a stable library.

**Comment by redcap3000** [comment]
In all seriousness I was actually fired for critical errors such as these long long ago while working in the (Silicon) Valley; but that was not a programming job. This is merely a hobby for me, I appreciate the attention to detail, language is something I struggle with daily. I have since banished myself from civilization as I am essentially unemployable in every industry.

**Comment by iamvinny** [comment]
It was totally unusable before, thank god it got fixed, good job @stackola.

**Comment by CAYdenberg** [comment]
No one's said it yet ...

298ay3

**Comment by redcap3000** [comment]
Attention grammar police; there is a new PR that desperately needs your attention, I feel if we all work together we can make these new changes a reality. https://github.com/redcap3000/crypto-socket/pull/9  ; I am very suspicious of the changes and require your assistance to validate them. I am somewhat opposed to the use of the oxford comma in technical documentation but I am open to the discussion so long as there are sufficient reviewers. While you're here please star the repo; perhaps it may reach trending status so that we may garner additional attention for this fastidious work.
